content/balsnctf_2023/Web3.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Web3
+date: 2023-10-11
+tags:
+  - misc
+author: expnx
+---
+
+# Web3
+
+Description: Hello Web3!
+
+Challenge Author: ysc
+
+we only have a url with port 3000
```

Comment:
This is also a stylistic choice, but I just want to note that you're using lowercase for everything. Not that important, just letting you know other writeups have capitalized sentences.